src/audio/copier/CMakeLists.txt

```diff
@@ -1,2 +1,2 @@
-add_local_sources(sof copier.c copier_hifi.c copier_generic.c)
+add_local_sources(sof copier.c copier_hifi.c copier_generic.c copier_ipcgtw.c)
```

Comment:
copier_ipc_gtw.c should be added with a dependency on CONFIG_IPC4_GATEWAY just like above right? Why are we removing it?